traffic_ops/testing/api/v2/servers_test.go

```diff
@@ -101,17 +101,23 @@ func UpdateTestServers(t *testing.T) {
 	hostName := firstServer.HostName
 	// Retrieve the server by hostname so we can get the id for the Update
 	resp, _, err := TOSession.GetServerByHostName(hostName)
+	originalHostname := resp[0].HostName
```

Comment:
We should probably do this after we check that the error is nil. In case we dont get back a valid resp, this can throw a seg fault due to dereferencing of a nil ptr.

Reply:
Sounds good. Changed it.